mesos-client/src/main/scala/mesosphere/mesos/examples/UselessFramework.scala

```diff
@@ -0,0 +1,62 @@
+package mesosphere.mesos.examples
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, Source}
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.mesos.client.MesosClient
+import mesosphere.mesos.conf.MesosConf
+import org.apache.mesos.v1.mesos.{Filters, FrameworkInfo}
+import org.apache.mesos.v1.scheduler.scheduler.Event
+
+import scala.util.{Failure, Success}
+
+object UselessFramework extends App with StrictLogging {
```

Comment:
Should call it ProudFramework AMIRITE? Too proud to accept any offers.

Reply:
NarcissisticFramework should also work :)